ansible/roles/druid-ingestion/templates/rollup_ml_survey_status

```diff
@@ -0,0 +1,243 @@
+{
+  "type": "kafka",
+  "spec": {
+    "ioConfig": {
+      "type": "kafka",
+      "consumerProperties": {
+        "bootstrap.servers": "{{kafka_brokers}}"
+      },      
+      "topic": "{{env}}.ml.survey.raw",
+      "taskCount":"{{ml-survey-status-rollup_taskcount}}",
```

Comment:
Same with this variable - Add this variable with default value under defaults/main.yml